courses/models.py

```diff
@@ -14,6 +14,7 @@ class Program(models.Model):
     title = models.CharField(max_length=255)
     live = models.BooleanField(default=False)
     description = models.TextField(blank=True, null=True)
+    personalized_pricing = models.BooleanField(default=False)
```

Comment:
yes, I was talking about that. I just merged my PR that contains already the field and the migration.